dissect/target/plugins/os/windows/wer.py

```diff
@@ -0,0 +1,163 @@
+        "sysvol/ProgramData/Microsoft/Windows/WER/ReportQueue",
+        "%userprofile%/AppData/Local/Microsoft/Windows/WER/ReportArchive",
+        "%userprofile%/AppData/Local/Microsoft/Windows/WER/ReportQueue",
+    ]
+
+    def __init__(self, target: Target):
+        super().__init__(target)
+        self.wer_files = [
+            res.iterdir()
+            for wer_dir in self.WER_LOG_DIRS
+            for res in self.target.fs.path(wer_dir).rglob("*")
+            if res.is_dir()
+        ]
+
+    def check_compatible(self) -> None:
+        if self.wer_files:
+            return
+        raise UnsupportedPluginError("No Windows Error Reporting directories found.")
+
+    @export(record=DynamicDescriptor(["path", "string", "datetime"]))
+    def wer(self) -> Iterator[DynamicDescriptor]:
+        """Return information from Windows Error Reporting (WER) files.
+
+        Windows Error Reporting (WER) is used by Microsoft to create a report when an application crashes. These reports
+        can be sent to Microsoft, on which basis Microsoft can provide the user with troubleshooting information. Since
+        malware usually crashes more often than legitimate software, the presence of these WER files and/or the
+        information within may be useful for analysis. For example, it may contain the file hash of the crashed
+        application within the target_app_id field.
+
+        Sources:
+            - https://learn.microsoft.com/en-us/windows/win32/wer/windows-error-reporting
+            - https://medium.com/dfir-dudes/amcache-is-not-alone-using-wer-files-to-hunt-evil-86bdfdb216d7
+
+        Yields dynamically created records based on the fields in the files. A record at least contains the following
+        fields:
+            wer_file_path (path): File path to the WER report on the target.
+            version (string): WER file version.
+            event_type (string): WER file event type.
+            event_time (datetime): The moment in time when the error event took place.
+            consent (string): WER file consent to be sent to Microsoft.
+            report_identifier: WER file report identifier
+            app_session_guid: GUID for the app session causing/reporting the error.
+            target_app_id: WER file target app ID which may contain the application hash.
+            target_app_ver: WER file target app version.
+            boot_id: WER file boot ID.
+            response_type: WER file response type.
+            friendly_event_name: Human readable event name.
+            app_name: WER file application name.
+            app_path: Path to application that caused/reported the error.
+            report_description: WER file report description.
+            application_identity: WER file application identity.
+            metadata_hash: WER file metadata hash.
+        """
+        for files in self.wer_files:
+            wer_fields = []
+            wer_values = {"_target": self.target}
```

Comment:
```suggestion
            record_values = {"_target": self.target}
```
I think `record_values` is a bit more accurate.